installation failed

when I ran `pipx install kmax`, got the following error

```
DEPRECATION: Building 'kmax' using the legacy setup.py bdist_wheel mechanism, which will be removed in a future version. pip 25.3 will enforce this behaviour change. A possible replacement is to use the standardized build interface by setting the `--use-pep517` option, (possibly combined with `--no-build-isolation`), or adding a `pyproject.toml` file to the source tree of 'kmax'. Discussion can be found at https://github.com/pypa/pip/issues/6334
```

- Python 3.13.5
- pip 25.3
- pipx 1.7.1
- gcc (Debian 14.2.0-19) 14.2.0
- Debian clang version 19.1.7 (3+b1)

okay. Above is just a warning. The actual error is
```
      kextractors/kextractor-3.19/bconf.tab.c:1879:7: error: call to undeclared function 'bconferror'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       1879 |       yyerror (YY_("syntax error"));
      kextractors/kextractor-3.19/bconf.tab.c:65:25: note: expanded from macro 'yyerror'
         65 | #define yyerror         bconferror
      kextractors/kextractor-3.19/bconf.tab.c:2023:3: error: call to undeclared function 'bconferror'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
       2023 |   yyerror (YY_("memory exhausted"));
      kextractors/kextractor-3.19/bconf.tab.c:65:25: note: expanded from macro 'yyerror'
         65 | #define yyerror         bconferror
      kextractors/kextractor-3.19/bconf.tab.c:2064:1: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
      kextractors/kextractor-3.19/bconf.tab.c:2107:1: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
      kextractors/kextractor-3.19/bconf.tab.c:2119:1: error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
       2119 | bconferror(char *msg) {
      4 warnings and 14 errors generated.
      error: command '/usr/bin/clang' failed with exit code 1
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: failed-wheel-build-for-install
```

--- a/kextractors/kextractor-3.19/bconf.tab.c
+++ b/kextractors/kextractor-3.19/bconf.tab.c
@@ -118,6 +118,7 @@ struct linked_list {
   void *data;
 };
 
+void bconferror(char *msg);
 
 void
 add_config_var(char *var, enum symbol_type stype, char *def, struct linked_list *dep_list)
@@ -2061,7 +2062,7 @@ yyreturn:
 
 char *filename;
 
-bconf_parse(char *file)
+void bconf_parse(char *file)
 {
 	struct symbol *sym;
 	int i;
@@ -2104,7 +2105,7 @@ bconf_parse(char *file)
 	/* sym_set_change_count(1); */
 }
 
-bconf_test_lexer(char *file)
+void bconf_test_lexer(char *file)
 {
   int t;
 
@@ -2116,7 +2117,7 @@ bconf_test_lexer(char *file)
   }
 }
 
-bconferror(char *msg) {
+void bconferror(char *msg) {
   fprintf(stderr, "error:%s:%d: %s\n", filename, bconflineno, msg);
   exit(1);
 }

--- a/kextractors/kextractor-3.2/bconf.tab.c
+++ b/kextractors/kextractor-3.2/bconf.tab.c
@@ -118,6 +118,7 @@ struct linked_list {
   void *data;
 };
 
+void bconferror(char *msg);
 
 void
 add_config_var(char *var, enum symbol_type stype, char *def, struct linked_list *dep_list)
@@ -2061,7 +2062,7 @@ yyreturn:
 
 char *filename;
 
-bconf_parse(char *file)
+void bconf_parse(char *file)
 {
 	struct symbol *sym;
 	int i;
@@ -2104,7 +2105,7 @@ bconf_parse(char *file)
 	/* sym_set_change_count(1); */
 }
 
-bconf_test_lexer(char *file)
+void bconf_test_lexer(char *file)
 {
   int t;
 
@@ -2116,7 +2117,7 @@ bconf_test_lexer(char *file)
   }
 }
 
-bconferror(char *msg) {
+void bconferror(char *msg) {
   fprintf(stderr, "error:%s:%d: %s\n", filename, bconflineno, msg);
   exit(1);
 }

--- a/kextractors/kextractor-4.12.8/bconf.tab.c
+++ b/kextractors/kextractor-4.12.8/bconf.tab.c
@@ -118,6 +118,7 @@ struct linked_list {
   void *data;
 };
 
+void bconferror(char *msg);
 
 void
 add_config_var(char *var, enum symbol_type stype, char *def, struct linked_list *dep_list)
@@ -2061,7 +2062,7 @@ yyreturn:
 
 char *filename;
 
-bconf_parse(char *file)
+void bconf_parse(char *file)
 {
 	struct symbol *sym;
 	int i;
@@ -2104,7 +2105,7 @@ bconf_parse(char *file)
 	/* sym_set_change_count(1); */
 }
 
-bconf_test_lexer(char *file)
+void bconf_test_lexer(char *file)
 {
   int t;
 
@@ -2116,7 +2117,7 @@ bconf_test_lexer(char *file)
   }
 }
 
-bconferror(char *msg) {
+void bconferror(char *msg) {
   fprintf(stderr, "error:%s:%d: %s\n", filename, bconflineno, msg);
   exit(1);
 }

